### PR TITLE
Same Ylim everywhere, faster RMS, png background fix

### PR DESCRIPTION
**Pull request by FMassin**

### CODE REVIEW

File: seismosocialdistancing.py [hunk 1]
```diff
@@ -590,14 +590,15 @@ def plot(displacement_RMS,
             cmap = plt.get_cmap("tab20")
 
             if type in ['*', 'all', 'dailyplots']:
-                ax = stack_wday_time(preloc).plot(figsize=(14,8), cmap = cmap)
-                stack_wday_time(postloc).plot(ls="--", ax=ax, legend=False,cmap = cmap)
+                ax = stack_wday_time(preloc,scale).plot(figsize=(14,8), cmap = cmap)
+                stack_wday_time(postloc,scale).plot(ls="--", ax=ax, legend=False,cmap = cmap)
                 
                 plt.title("Daily Noise Levels in %s" % (channelcode[:]+main[-1]))
                 plt.ylabel("Amplitude (nm)")
```
[the remark this answers is not in the document]
Comment (ThomasLecocq) [reply seismosocialdistancing.py:674]: @FMassin idea to tackle this?

[the remark this answers is not in the document]
Comment (FMassin) [reply seismosocialdistancing.py:674]: Yes, we should allow unit to be an argument too for consistency...

[the remark this answers is not in the document]
Comment (FMassin) [reply seismosocialdistancing.py]: Done in next commit, works on my server :)

**Comment by ThomasLecocq** [comment]
testing now ...

**Comment by ThomasLecocq** [comment]
merged,... have to say i'm a bit lost in the different drms, df_rms, etc function names... but for now it works!

**Pull request merged**